apps/web/next.config.mjs

```diff
@@ -190,6 +190,11 @@ const config = {
         destination: 'https://tokens.pancakeswap.finance/images/:address',
         permanent: false,
       },
+      {
+        source: '/trading-reward',
+        destination: '/trading-reward/top-traders',
+        permanent: true,
```

Comment:
are we going to have page url set to '/trading-reward' in the future?

Reply:
From Bun
Better to keep the root while having other two as dedicated